src/longopt/gmtselect_inc.h

```diff
@@ -24,36 +24,26 @@ static struct GMT_KEYWORD_DICTIONARY module_kw[] = {
 	/* separator, short_option, long_option,
 	          short_directives,    long_directives,
 	          short_modifiers,     long_modifiers */
-	{ 0, 'A', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'C', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'D', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'E', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'L', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'F', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'G', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'I', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'N', "",
-	          "",                  "",
-	          "",                  "" },
-	{ 0, 'Z', "",
-	          "",                  "",
-	          "",                  "" },
+	{ 0, 'A', "min_area|area",
+	          "",                  "",
+	          "a,l,r,p",           "antarctica,lakes,river_lakes,percent" },
+	{ 0, 'C', "distance|dist2pt",
+	          "",                  "",
+	          "d",                 "distance" },
+	{ 0, 'D', "resolution",
+	          "f,h,i,l,c",         "full,high,intermediate,low,crude",
+	          "f",                 "lower" },
+	{ 0, 'E', "boundary",          "", "", "", "" },
+	{ 0, 'F', "polygon",           "", "", "", "" },
+	{ 0, 'G', "grid_mask",         "", "", "", "" },
+	{ 0, 'I', "reverse|invert",    "", "", "", "" },
+	{ 0, 'L', "dist2line",
+	          "",                  "",
+	          "d,p",               "distance,project" },
+	{ 0, 'N', "feature_mask",      "", "", "", "" },
+	{ 0, 'Z', "z_range",
```

Comment:
Why `z_` in `z_range`? I had it `in_range` in Julia and don't think it's worse.

Reply:
grdlandmask has a **-N** option similar to **gmtselect**. It talks about `maskvalues`.  If we have to have the word "mask" in the long option then it remains unclear (what are we masking?).  Maybe use `--inout` ? Perhaps the silent majority of @GenericMappingTools/long-options-review have some suggestions? We should use the same phrase for both **gmtselect** and **grdlandmask**.  Then, **grdmask** is a bit different, perhaps, but `--inout` might work there too.

As for **-Z** I also think we don't want "z" in the name since can be any column.  And `range` is used for other things (e.g., `--range|series` for the many **-T** array options).  Maybe `--limits`? I think we restrict ourselves too much if we want a long-option name that starts with the short-option letter (here Z).

Reply:
Please note that range is a Python keyword, so PyGMT can't use it.

Reply:
We will minimize its use

Reply:
Finalised to do mask|maskvalues for -N, invert|reverse for -I, limits|in_range for -Z.